Resources/Prototypes/_White/Entities/Structures/Storage/SenorsLockers.yml

```
@@ -39,7 +39,7 @@
         prob: 0.5
       - id: ClothingEyesGlassesSecurity
       - id: ClothingHeadsetAltSecurity
-      - id: ClothingHandsGlovesColorBlack
+      - id: ClothingHandsGlovesColorBlackSecurity # WD
```

yamllint (1.35.1)

[warning] too few spaces before comment

(comments)